### DISCUSSION TITLE
meta tables shown in UI [Apr 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1993] [Discord Threads]
> 🙋‍♂️ **Xpressd**  
#### are we supposed to be seeing these 33 tables on the UI ?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/969064437589893170/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### thanks for reporting. previously they were not shown. there may be something wrong with the filtering part. It is not supposed to see them. we'll make the fix soon. here's the issue for the record - https://github.com/nocodb/nocodb/issues/1910.  
#  
> 👨‍🚀 **wingkwong**  
#### btw may i know which version r u using?  
#  
> 🙋‍♂️ **Xpressd**  
#### do you know where i can check my version?    
####    
#### I installed a fresh nocodb last night by running a new docker container, doesnt it automatically grab the most recent version?   
####    
#### on Nocodb it says there is an upgrade available and it says something like click here to view the 0.97 changelog, does that indicate the version?    
####    
#### does that mean I am I at 0.97?  
#  
> 👨‍🚀 **wingkwong**  
#### i believe u mean 0.90.7  
#  
> 👨‍🚀 **wingkwong**  
#### it s ok. the issue has been fixed n will be available in next release  
#  
> 👨‍🚀 **wingkwong**  
#### however, it's recommended to separate your data database and your meta database. see https://docs.nocodb.com/engineering/architecture  
#  
> 🙋‍♂️ **Xpressd**  
#### ok great, I will read about the architecture. thanks for looking into this  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969064438147739668)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
